README.md

```diff
@@ -1,29 +1,34 @@
 # OSIE
 
 [![Build Status](https://drone.packet.net/api/badges/tinkerbell/osie/status.svg)](https://drone.packet.net/tinkerbell/osie)
-![](https://img.shields.io/badge/Stability-Experimental-red.svg)
+[![deprecated](http://badges.github.io/stability-badges/dist/deprecated.svg)](http://github.com/badges/stability-badges)
```

Comment:
Why not just change the Experimental -> Deprecated? The new badge has a completely different style.

```suggestion
![deprecated](https://img.shields.io/badge/Stability-Deprecated-red.svg)
```